src/hypothesis/strategies.py

```diff
@@ -910,6 +912,108 @@ def do_draw(self, data):
     return PermutationStrategy()
 
 
+@defines_strategy
+def datetimes(min_datetime=dt.datetime.min, max_datetime=dt.datetime.max,
+              timezones=None):
```

Comment:
Would it be better to have `timezones` default to `none()` and not allow None as a value here? That way it's always consistently a strategy object.